src/mbgl/map/map.cpp

```diff
@@ -54,6 +56,7 @@ class Map::Impl : public style::Observer {
 
     RenderState renderState = RenderState::Never;
     Transform transform;
+    std::unique_ptr<CameraOptions> initialCameraOptions;
```

Comment:
`CameraOptions` is just a struct, so there’s no need to have a pointer to it. It looks like you’re using `std::unique_ptr` as a way of allowing the options to be null. In that case, use `optional` instead.